lib/sinon/behavior.js

```diff
@@ -126,6 +127,8 @@ var proto = {
             return args[this.returnArgAt];
         } else if (this.returnThis) {
             return context;
+        } else if (typeof this.throwArgAt === "number") {
+            throw args[this.throwArgAt];
```

Comment:
I think this should test whether the function was called with enough arguments to thew the argument at the given index.